src/core/AutoRest/AutoRest.json

```diff
@@ -35,6 +35,9 @@
     },
     "AzureResourceSchema": {
       "type": "AzureResourceSchemaCodeGenerator, AutoRest.AzureResourceSchema"
+    },
+    "Go": {
```

Comment:
>Go [](start = 5, length = 2)

Should this be `Azure.Go` or does it support generating generic Go SDK generation?